### ISSUE TITLE
Missing Dialogue to modify Valid Contexts for a Macro

### ISSUE THREAD

**Issue by m-kovalsky** [TabularEditor/TabularEditor3#189]
After saving a macro, I don't see a way to modify the 'Valid Contexts' of the macro (as is possible with Custom Actions in TE2). In the Macros toolbar, it's only possible to change the name of the Macro, not the Valid Contexts. Perhaps there's another way but I don't seem to be able to find it.

Thanks!

**Comment by samaguire** [comment]
If you open the macro (double clicking the 'id' or 'valid contexts' columns), you can then update the applicable contexts by clicking on the highlighted toolbar button below.

NB: Under advance options you can control if a macro even displays or not as an action. (Though probably easier just to allow no context selection 🤷‍♂️)

BTW - loving your scripts 👍

![image](https://user-images.githubusercontent.com/62320770/130733197-3f1d18c2-3270-4ba9-a518-a25d124c7e27.png)

**Comment by m-kovalsky** [comment]
Ah thanks Stephen.

However, I don't seem to get the 'OK/Cancel' portion of the window. And, this is when TE3 is open on my laptop's screen (not a monitor) - see the screenshot below.

Also, it would be nice to be able to right click on a Macro within the Macros toolbar and have the option to edit the properties (and not having to first open the macro itself). I guess the experience of editing a macro feels a bit unintuitive (but could just be my opinion).

![image](https://user-images.githubusercontent.com/29556918/130785661-96510382-3f2f-4783-a9d2-10ba799cc4db.png)

**Comment by otykier** [comment]
You’re absolutely right @m-kovalsky - there is definitely room for improvements here.

**Comment by m-kovalsky** [comment]
I'm actually not able to reproduce the issue with the OK/Cancel buttons not appearing (now OK/Cancel shows up fine every time on my main screen). The meta point here is that one shouldn't have to set customized settings for something like this to work.

**Comment by otykier** [comment]
@m-kovalsky I agree. The cause of this bug is the UI scaling issues that have been discussed before (see #177, #81). We're still awaiting a fix from our UI vendor.

**Comment by mlonsk** [comment]
I believe these UI issues have been solved in newer versions of TE3, but please let us know if they still occur.
